### PR TITLE
P0 alert button components

### PR DESCRIPTION
**Pull request by dhimanaishwarya**
Alert Components
Button Components

### CODE REVIEW

**Review by anshulraheja — APPROVED** [review]
Code is good to merge.

File: app.css [hunk 1]
```diff
@@ -111,9 +110,31 @@ a:hover {
...
+  display: flex;
+}
+
+.component-div-column {
+  flex-direction: column;
+}
+
+.code-iframe {
+  width: 790px;
+  height: 262px;
```
Comment (ashishkatiyar2404) [inline app.css]: use px when <10px. If  you declare it for  purpose  than ok.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.alert-success {
+  background: var(--GREEN-COLOR-LIGHT);
+  color: var(--GREEN-COLOR-DARK);
+}
+
+.alert-warning {
+  background: var(--YELLOW-COLOR-LIGHT);
+  color: var(--ZINC-COLOR-DARK);
+}
```
Comment (ashishkatiyar2404) [inline components/alert/alert.css:37]: good naming convention.

Comment (ashishkatiyar2404) [reply components/alert/alert.css:37]: just a suggestion. don't use long naming for root var.
you can give like:- ZINZ-DARK,  YELLOW-LIGHT.

**Review by ashishkatiyar2404 — APPROVED** [review]

**Comment by ashishkatiyar2404** [comment]
code looks good. Ready to merge💯

**Pull request merged**